internal/commands/set_registry.go

```diff
@@ -0,0 +1,54 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+	"gopkg.in/errgo.v2/fmt/errors"
+
+	"github.com/buildpacks/pack/internal/config"
+	"github.com/buildpacks/pack/internal/style"
+	"github.com/buildpacks/pack/logging"
+)
+
+func SetRegistry(logger logging.Logger, cfg config.Config, cfgPath string) *cobra.Command {
+	var (
+		registryName string
+	)
+
+	cmd := &cobra.Command{
+		Use:   "set-registry <name>",
```

Comment:
Based on already having `set-default-builder`, this should be `set-default-registry` for symmetry.

Reply:
I like that